cloudwatch-snowflake/README.md

```diff
@@ -0,0 +1,40 @@
+# Terraform Module: Cloudwatch Snowflake
+
+Syncing the following Cloudwatch metrics to Snowflake table thru Fivetran:
+
+- CPUUtilization
+- MemoryUtilization
+- TargetResponseTime (p50, p90, p99)
+- 5xx Error Count
+
+```
+module "metrics" {
+  source = "github.com/dbl-works/terraform//cloudwatch-snowflake"
+
+  fivetran_group_id = "fivetran-group-id"
+  organisation = "meta"
+  fivetran_api_key = XXXXXXXXXXX
+  fivetran_api_secret = YYYYYYYYYY
+  tracked_resources_data = [
+    {
+      projectName      = "facebook"
+      serviceName      = "web",
+      clusterName      = "facebook-production"
+      loadBalancerName = "app/facebook-production/7ed111aa2bf1de37"
```

Comment:
- how do we obtain the value?

Reply:
This is the `aws_lb => arn_suffix`. 
Have updated the comment :D